wandb/sdk/launch/builder/noop.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Dict, Optional
+
+from wandb.errors import LaunchError
+from wandb.sdk.launch.builder.abstract import AbstractBuilder
+
+from .._project_spec import EntryPoint, LaunchProject
+
+
+class NoOpBuilder(AbstractBuilder):
+
+    type: str
```

Comment:
woah you can add typing in a class def like this? cool.

Reply:
presumably returns when asked for __type?

Reply:
Hmm, I'm not sure how I added this as str. Should be "noop" nice catch